src/Microsoft.Health.Fhir.Shared.Tests/Samples.cs

```diff
@@ -96,6 +97,11 @@ public static ResourceElement GetDefaultConvertDataParameter()
             return GetJsonSample("Parameter-Convert-Data");
         }
 
+        public static ImportRequest GetDefaultBulkImportRequest()
```

Comment:
It doesn't look like this sample is used in any tests. Can it be removed?

Reply:
Yes, it is not used.